web/themes/weathergov_admin/weathergov_admin.theme

```diff
@@ -52,24 +60,65 @@ function weathergov_admin_form_node_form_alter(
             ->isPublished();
 
         if ($isPublished) {
-            $weight = 1000;
-            if (array_key_exists("delete", $form["actions"])) {
-                if (array_key_exists("#weight", $form["actions"]["delete"])) {
-                    $weight = $form["actions"]["delete"]["#weight"] - 1;
-                }
-            }
-
             $form["actions"]["unpublish"] = [
                 "#type" => "submit",
                 "#value" => "Unpublish",
                 "#name" => "unpublish",
                 "#submit" => ["weathergov_form_node_unpublish"],
                 "#weight" => $weight,
             ];
+        } else {
+            $form["actions"]["publish"] = [
+                "#type" => "submit",
+                "#value" => "Publish",
+                "#name" => "publish",
+                "#submit" => ["weathergov_form_node_publish"],
+                "#weight" => $weight,
+                "#attributes" => ["disabled" => !$isPreviewed],
+            ];
         }
+    } else {
+        $form["actions"]["publish"] = [
+            "#type" => "submit",
+            "#value" => "Publish",
+            "#name" => "publish",
+            "#submit" => ["weathergov_form_node_publish"],
+            "#weight" => $weight,
+            "#attributes" => ["disabled" => !$isPreviewed],
+        ];
+    }
+
+    if (!$isPreviewed) {
+        $form["previewRequirement"]["notice"] = [
+            "#type" => "markup",
+            "#markup" =>
+                "Content must be previewed before it can be saved or published.",
```

Comment:
I wonder if we should update this for the "Save updates as draft" flow to include some help text around how one gets a new version of a draft published... is it going to "Latest version" and changing the moderation state?